folium/plugins/timedynamic_geo_json.py

```diff
@@ -0,0 +1,257 @@
+import json
+from ..map import Layer
```

Comment:
Please use absolute imports to keep it consistent with the rest of the code.

Reply:
OK